File: tests/expectations/typedef.zig
const std = @import("std");

pub const Foo_i32__i32 = extern struct {
  x: i32,
  y: i32,
};

pub const IntFoo_i32 = Foo_i32__i32;

extern fn root(a: IntFoo_i32) anyopaque;

File: tests/expectations/both/typedef.c
#include <stdint.h>
#include <stdlib.h>
#include <stdbool.h>

typedef struct Foo_i32__i32 {
  int32_t x;
  int32_t y;
} Foo_i32__i32;

typedef Foo_i32__i32 IntFoo_i32;

void root(IntFoo_i32 a);
